packages/cli/src/commands/pg/settings/auto-explain/log-min-duration.ts

```diff
@@ -12,7 +12,7 @@ export default class LogMinDuration extends PGSettingsCommand {
 
   static args = {
     database: Args.string(),
-    value: Args.integer(),
```

Comment:
Does this work with the way that `explain` is defined? Because that's expecting the setting value to be a number. If it does I guess that's ok? Although the value really is supposed to be an integer, so we're losing a little bit of input validation here by defining it as `string`.

Reply:
Good thought. In fact, it does work. I'm adding a screenshot here so you can see that, by the time the `setting` object hits the `explain` function, `setting.value` is a number (I added a console.log to print out the type). 
<img width="997" alt="image" src="https://github.com/user-attachments/assets/1388623c-13a5-490c-a0f7-c13cb1be841d">

I did go through the exercise of trying to make this work while leaving args.value as an integer, but unfortunately because of a bug in the oclif/core v2 code, we cannot use an integer type here if we are expecting a value of zero. When we are able to upgrade to oclif/core v3 or above, this will be fixed and we should be able to use an integer type here.

The problem is this [little bit of logic](https://github.com/oclif/core/blob/f87fbdbb7ac206ae6229c3836abfd7c8dd6ddfcd/src/parser/parse.ts#L523-L526) inside oclif's parser functionality. When the arg is zero, the `if (args[token.arg])` returns as falsy. At that point, it pushes "0" into the argv array, which already contains the original zero, when it should hit that continue.

Reply:
Nice find, if `integer` is broken for our edition of ocif, let's change https://github.com/heroku/cli/blob/main/packages/cli/src/commands/pg/settings/log-min-duration-statement.ts as well.